packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -100,25 +103,28 @@ export default class TelecomPackMigrationOffersCtrl {
   static updateSelectedGtrOption(offer, optionName) {
     let optionComfort = false;
     Object.values(offer.options).forEach((option) => {
-      if (
-        option.name.startsWith('gtr_') &&
-        option.selected !== null &&
-        optionName.startsWith('gtr_')
-      ) {
-        if (option.name !== optionName) {
-          set(option, 'selected', false);
+      if (option.name.startsWith('gtr_') && option.selected !== null) {
+        if (optionName.startsWith('gtr_')) {
```

Comment:
**nitpick:** Checking same condition which is already present in the above `if` is redundant
```suggestion
       
```

Reply:
No it's not the same condition, first is `option.name`  which is tested, second is `optionName` which is tested